kingpin/bin/deploy.py

```diff
@@ -236,7 +236,7 @@ def begin():
         # Skip traceback that involves tornado's libraries.
         import traceback
 
-        trace_lines = traceback.format_exc(e).splitlines()
+        trace_lines = traceback.format_exc().splitlines()
```

Comment:
This is the only real change in this PR. Everything else is just formatting required by linter.